### ISSUE TITLE
How to use in CPPRest SDK in Cross Platform Shared Library Project, Visual Studio 2015.

### ISSUE THREAD

**Issue by Gurdeep0602** [Microsoft/cpprestsdk#268]
I need to to create **Cross Platform Shared Library in Visual Studio 2015**, it will serve as the common codebase for both iOS and android apps which i'll be building using this library. I've been able to successfully use CPPRest SDK in an empty project in Visual Studio via Nuget Package Manager but when i create a Shared Lib project and install CPPRest SDK via Nuget, it does not allow me to use it. (Doesn't let me include header, specifically). 

I believe I'll have to build CPPRest SDK for [iOS](https://casablanca.codeplex.com/wikipage?title=Setup%20and%20Build%20on%20iOS) and [Android](https://casablanca.codeplex.com/wikipage?title=Use%20on%20Android) separately and then place them in their respective directories in the project but I'm not sure how will it work exactly as building for iOS contains inclusion for native iOS libraries also, how will they be placed in Visual Studio. 

Can anyone please help ? Has anyone done similar kind of thing ?

**Comment by ras0219-msft** [comment]
Yes, you will need to build the library for each platform separately first.

Generally, you will need to add the built libraries for each platform to the platform-specific projects and the shared include directory to all the projects.

**Comment by Gurdeep0602** [comment]
I've built the ".a" file for CPPRest SDK, and included headers for openssl, cpprest and boost in iOS directory of the project. But I'm stuck on android part, on how to include headers.

I'm not able to write code currently, which uses cpprest in shared directory of the project. (i guess, because android does not understand it). Any help will be useful. Thanks!
